Cargo.lock

```diff
@@ -3200,7 +3256,7 @@ dependencies = [
  "rocksdb",
  "serde",
  "serde_json",
- "strum",
+ "strum 0.20.0",
```

Comment:
Can we upgrade `strum` to avoid duplication? Might to a pre-PR for that.

Reply:
Out today, but will do.

Reply:
FYI, I have some WIP PRs that clean up and update strum. Currently they are waiting for #6678 but I can speed them along if you want.

Reply:
Would it be that jarring of a conflict if this merged first? Then again, I don't mind holding this until you're set.

Reply:
Nope. I’m fine either way. Updating to strum 0.24 requires some cleanups or `#[allow(deprecated)]`.

Reply:
General observation: if the crate is at version `0.xx`, it's a medium evidence that using such crate means accepting a bit of churn.

Reply:
Actually nvm, I was being a dummy. There’s `strum::IntoStaticStr` we should be migrating to.  I’ll have a PR soon.

Reply:
I got strum to the state where `sed -i '/^strum/ s/0.20/0.24/' -- $(git grep -l strum)` will work for the most part. This, however, brings some issue with duplicate strum_macros dependency. This could be resolved by your change though.  Either way, I spent way too much time on this already, so I’ll let you deal with the dependency hell. ;)

Reply:
> Can we upgrade strum to avoid duplication? 

Done